.github/workflows/builds.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
           ./bin/build-addon.sh nightly.xpi
 
       - name: Uploading
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
Why?

Reply:
do you refactor or implement an isolated feature?

Reply:
It's a feature, but as I remember this`v3` was causing issues with actions.

Reply:
Fun fact. After rebasing on `main`, there's no difference anymore. 
`v4` is already fixed in the main branch some time ago